assets/js/components/data-block.js

```diff
@@ -87,11 +87,7 @@ class DataBlock extends Component {
 		if ( change ) {
 			// If changeDataUnit is given, try using it as currency first, otherwise add it as suffix.
 			if ( changeDataUnit ) {
-				try {
-					changeFormatted = numberFormat( Math.abs( change ), { style: 'currency', currency: changeDataUnit } );
-				} catch ( e ) {
-					changeFormatted = `${ numberFormat( Math.abs( change ) ) }${ changeDataUnit }`;
-				}
+				changeFormatted = numberFormatWithUnit( Math.abs( change ), changeDataUnit );
```

Comment:
See above, let's make this
```suggestion
				changeFormatted = readableLargeNumber( Math.abs( change ), changeDataUnit );
```